targets/x11-common

```diff
@@ -7,6 +7,12 @@
 # forms of X11.
 
 ### Append to prepare.sh:
+
+#we need to install xinit to be sure we overwrite the path
```

Comment:
what does this mean

Reply:
TBH I can't remember, but we need to install xinit, otherwise graphic does not work.
Do you prefer that I add this line instead of the if:
install opensuse=xinit,

Just let me know.

Reply:
I'd prefer to not install things unnecessarily, so please check what xinit is pulling in when you try to install it here

Reply:
It's not pulling in anything, no recommends and sugguests:

linux@localhost:~> zypper info --recommends --suggests xinit
Loading repository data...
Reading installed packages...


Information for package xinit:
------------------------------
Repository     : openSUSE-Ports-Leap-42.3-repo-oss
Name           : xinit                            
Version        : 1.3.4-3.1                        
Arch           : aarch64                          
Vendor         : openSUSE                         
Installed Size : 234.6 KiB                        
Installed      : Yes (automatically)              
Status         : up-to-date                       
Source package : xinit-1.3.4-3.1.src              
Summary        : X Window System initializer      
Description    :                                  
    The xinit program is used to start the X Window System server and a
    first client program on systems that are not using a display manager
    such as xdm or in environments that use multiple window systems.
    When this first client exits, xinit will kill the X server and then
    terminate.
Recommends     : ---                              
Suggests       : ---

Reply:
Ah, I see.  This is fine, then.  Just move it under the "remove old vgem hack" section and quote $DISTROAKA

Reply:
Oh weird, we're already installing xinit in the xorg.  Is this an issue for xorg or only for xiwi?

Reply:
...in which case, xiwi line 31 is probably the right place for it.

Reply:
ok, I put it at line 31 in xiwi.